multigeiger/multigeiger.ino

```diff
@@ -688,12 +688,12 @@ void DisplayStartscreen(void){
   u8x8.drawString(0, 2, "Geiger-");
   u8x8.drawString(0, 3, " Counter");
   char rv[9];
-  strncpy(rv, revString, 4);
-  rv[4] = '\0';
+  strncpy(rv, revString, 5);
+  rv[5] = '\0';
   u8x8.drawString(2, 4, rv);
-  strncpy(rv, &revString[5], 4);
-  strncpy(&rv[4], &revString[10], 2);
-  strncpy(&rv[6], &revString[13], 2);
+  strncpy(rv, &revString[6], 4);
+  strncpy(&rv[4], &revString[11], 2);
+  strncpy(&rv[6], &revString[14], 2);
```

Comment:
this is rather ugly and maybe should be solved in another way.

guess as it broke now (1.9 -> 1.10), it will break again (1.xx -> 2.0),

Reply:
You're right, that's ugly. Will be changed in one of the next releases